onnxruntime/contrib_ops/rocm/bert/tunable_op.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why this file `tunable_op.h` is placed in `bert` directory?

Reply:
The functionality is actually very general purpose and could potentially be used for other ops and EPs. I thought we can start with ROCM EP and BERT inference scenarios, so put it here.

Reply:
Since for now, we only have 4 main Ops in attention to take care of, keeping it in bert folder makes sense.